systemtest/containers.go

```diff
@@ -414,8 +414,7 @@ func (c *ElasticAgentContainer) Close() error {
 func (c *ElasticAgentContainer) APMServerLog() (io.ReadCloser, error) {
 	return c.container.CopyFileFromContainer(
 		context.Background(), fmt.Sprintf(
-			"/usr/share/elastic-agent/data/elastic-agent-%s/components/logs/apm-server-%s.ndjson",
-			c.vcsRef[:6],
+			"/usr/share/elastic-agent/state/data/logs/apm-default-%s-1.ndjson",
```

Comment:
Not exactly in the scope of this PR but the current path to apm server logs started by elastic agent is wrong